atlasdb-client-protobufs/build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
     mavenCentral()
   }
   dependencies {
-    classpath 'com.google.protobuf:protobuf-gradle-plugin:0.7.6'
```

Comment:
Woah, if any of the persisted files changed this cannot merge unless we can confirm that their on disk formats have not changed.  

That said, changing the gradle plugin shouldn't result in any changes to those files.